.github/workflows/e2e.yml

```diff
@@ -0,0 +1,84 @@
+name: E2E Test Profile
+
+on:
+  workflow_dispatch:
+
+    inputs:
+      o3_url:
+        description: 'O3 URL'
+        required: true
+      odoo_url:
+        description: 'Odoo URL'
+        required: false
+      senaite_url:
+        description: 'SENAITE URL'
+        required: false
+      superset_url:
+        description: 'Superset URL'
+        required: false
+      test_pro:
+        description: 'Running Ozone on Pro?'
+        required: true
+        default: 'true'
+        type: choice
+        options:
+          - true
+          - false
+      test_profile:
+        description: 'Choose test profile'
```

Comment:
Should we simplify this to `Choose Distro`
cc: @mks-d

Reply:
```code      
options:
        - ozone-pro
        - ozone-foss
        - openmrs-distro-his
  ```
@VaishSiddharth those are not actually distributions but rather profiles that run a subset of E2E test suite. Refer to https://github.com/ozone-his/ozone-e2e/pull/112/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R16-R20